pkg/pop/mix.cue

```diff
@@ -0,0 +1,49 @@
+import "encoding/json"
+
+// TODO: include full schema?
+// TODO: figure out why it stops working when uncommented (cue eval works tho)
+// prometheusAlerts: v0: {...}
+// prometheusRules: v0: {...}
+// grafanaDashboards: v0: {...}
+
+// TODO: handle different versions, deal with migrations, etc, etc
+mixin: {
```

Comment:
i haven't written a `_test.cue` file yet, but i think they're a thing, and this seems like an appropriate place to start...? :)

Reply:
derp. seems like i just totally imagined that.

well. `_test.go` files it is, then :D